email/resource_email.go

```diff
@@ -88,13 +100,33 @@ func resourceEmailCreate(d *schema.ResourceData, m interface{}) error {
 		from, []string{to}, []byte(msg))
 
 	if err != nil {
-		log.Printf("smtp error: %s", err)
-		return err
+		errorCode := extractStatusCode(err.Error())
+		// make tf configurable eventually? maybe?
+		maxRetries := 5
+		if errorCode == "421" {
```

Comment:
Can we combine this block with the block above to reduce code duplication?

Can be something like:
```
maxRetries = 5
for 1..maxRetries+1:
    err = smtp.SendMail(...)
    if err == nil:
        break
    errorCode = extractErrorCode(...)
    if errorCode != 421:
        break
    time.sleep(...)
```